linode/manager
# test: [M3-8458] - Fix StackScripts pagination test failure

## Description 📝
This fixes the StackScripts pagination test failure. The test fails because Cloud Manager filters certain StackScripts from the landing page, and our test didn't account for that when it checked for content in the StackScripts list.

Shout out to @AzureLatte for identifying the cause of the failure!

## Changes  🔄
- Do not assert that StackScripts exist in the list when they only support deprecated or Kube Images
- Rename `smoke-community-stackscrips.spec.ts` to `smoke-community-stackscripts.spec.ts`

## How to test 🧪
We can rely on CI since the test currently fails 100% of the time. However, you can run the test locally to manually verify the changes:

```bash
yarn cy:run -s "cypress/e2e/core/stackscripts/smoke-community-stackscripts.spec.ts"
```

## As an Author I have considered 🤔

*Check all that apply*
